R/adducts.R

```diff
@@ -39,12 +46,15 @@
 #'
 #' ## Calculate m/z of multiple adducts from neutral mass
 #' mass2mz(exact_mass, adduct = adductNames())
+#'
+#' ## Provide a custom adduct definition.
+#' adds <- data.frame(mass_add = c(1, 2, 3), mass_multi = c(1, 2, 0.5))
+#' rownames(adds) <- c("a", "b", "c")
+#' mass2mz(c(100, 200), adds)
 mass2mz <- function(x, adduct = "[M+H]+") {
-    idx <- match(adduct, names(.ADDUCTS_ADD))
```

Comment:
In case of custom adducts this will fail, or?

Reply:
Sure, but I removed these lines (that's why they are red) and replaced it with the (green) lines below.

Reply:
Man... too late... of course! ;-)